api/v1/helpers.go

```diff
@@ -1193,6 +1252,21 @@ func (v *VerticaDB) GetSandboxStatusCheck(sbName string) (*SandboxStatus, error)
 	return sb, nil
 }
 
+// DoesSandboxHaveQuorum returns true if the sandbox will keep quorum
+func (v *VerticaDB) DoesSandboxHaveQuorum(sbName string, offset int) bool {
```

Comment:
I don't fully understand this function. Can you add more details with examples.

Reply:
Example added.